### ISSUE TITLE
Create Beacon Data Audit Map

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#3649]
Ability to:
- create new beacons
- edit existing beacons
- mark beacons as deleted
- use Nearmap aerials basemap

Next steps:
- John to send schema
- Jaime to review schema with Billy. Ask if they want to do field checks/use Collector.

**Comment by johnclary** [comment]
Ability to:
- create new beacons
- edit existing beacons
- mark beacons as deleted
- use Nearmap aerials basemap

**Comment by johnclary** [comment]
@jaime-mckeown here's a list of the fields and controlled values we need for this layer. It also includes the sample data that is in the app today:
https://docs.google.com/spreadsheets/d/1NpPK1Icqpgfcfh17i2uZVNAzCBFwYX_eYa_kV-fpk4Y/edit#gid=1110700016

You can make a copy of the existing beacons layer and just hide/remove the other fields in that layer. Let me know if this doesn't make sense and I'm happy to talk more.

**Comment by jaime-mckeown** [comment]
Meeting with Billy Friday morning to go over the current fields that are in Knack.

**Comment by johnclary** [comment]
thanks @jaime-mckeown. good stuff. let me know if you need any help/input from me.

**Comment by jaime-mckeown** [comment]
Talked to John about the School Zone Beacon data after talking with Billy.
- Location address field is currently geocoded through HERE service which can give unusual results. We are going to allow the Signal Tech's to assign an address to each SZB, with the address fields broken up for cleaner editing (Street Number, Street Pre-Direction, Street Name, Street Type). Domains can be added on Street Pre-Direction and Street Type fields.
- John helped me figure out what the domain for Mount Location meant. Added to field description above, sent Billy email.

**Comment by jaime-mckeown** [comment]
PREFIX_DIRECTIONn
- E (East)
- N (North)
- S (South)
- W (West)

PREFIX_TYPE
- CR
- FM
- IH
- SH
- US

STREET_TYPE
- ALY (Alley)
- AVE (Avenue)
- BLVD (Boulevard)
- BND (Bend)
- BR (Branch)
- BRG (Bridge)
- CIR (Circle)
- CRES (Crescent)
- CT (Court)
- CTOF ?
- CV (Cove)
- CYN (Canyon)
- DR (Drive)
- EXPY (Expressway)
- GDNS (Gardens)
- GLN (Glen)
- HOLW (Hollow)
- HWY (Highway)
- LN (Lane)
- LOOP (Loop)
- PARK (Park)
- PASS (Pass)
- PATH (Path)
- PKWY (Parkway)
- PL (Place)
- PLZ (Plaza)
- PT (Point)
- RACE ?
- RAMP (Ramp)
- RD (Road)
- ROW (ROW)
- RUN (Run)
- SKWY (Skyway)
- SPUR (Spur)
- SQ (Square)
- ST (Street)
- SVRD (Service Road)
- TER (Terrace)
- TRCE (Trace)
- TRL (Trail)
- TRN ?
- VW (View)
- WALK (Walk)
- WAY (Way)
- XING (Crossing)

**Comment by jaime-mckeown** [comment]
Created a Spreadsheet to send to Billy so he can verify the fields that they will be modifying before I create the dataset in AGOL.

**Comment by jaime-mckeown** [comment]
- Billy said the fields were good.
- Created the feature class, included some of the fields he didn't need for future use and turned them off for the edit project.
- Created the School Zone Beacon Audit Map and configured. Added Bookmarks for each school to make it easier for the tech's to get around in the map.
- Created the School Zone Beacon Audit Web App for desktop editing.
- Checked the Web Map in Collector, changed Basemap to World Street Map as State Plane Maps don't draw in Collector.
- Created COA_ATD_School_Zone_Beacons_Audit Group in AGOL to ensure only the specified people can edit the data.

**Comment by jaime-mckeown** [comment]
- [ ] Go over Web App with Billy, get Tech names to add to Group.

- [ ] Training with Tech's to go over Editing in Web App and Collector.

**Comment by jaime-mckeown** [comment]
Added Installation Date to Feature Layer and configured Pop-up.

**Comment by jaime-mckeown** [comment]
Discussion with Billy about Audit Map
- He said that all the fields were good
- All data they needed to do the audit is included ( I can always add more layers if needed)
- We discussed the ability to add their own addresses and I shared with him the idea of CTM Addressing Group assigning physical addresses to the beacons. I'm waiting to hear back from Jacquie regarding that request.
- He will get with his staff to figure out exactly who will be starting work on this project.
- Schedule training for next week based on their schedules.
- I will need to add users to the AGOL Group to have access to the Web App.
